api/lib/application/passwords/index.js

```diff
@@ -35,7 +38,31 @@ exports.register = async function(server) {
         handler: passwordController.checkResetDemand,
         tags: ['api', 'passwords']
       }
-    }
+    },
+    {
+      method: 'POST',
+      path: '/api/expired-password-updates',
```

Comment:
Pourquoi il y a un "S" à updates ?

Reply:
Ember-data, cf. mon-pix/app/adapters/user.js
`const url = this.buildURL('expired-password-update', null, snapshot, 'createRecord');`

Reply:
Done